### PR TITLE
Automatically remove unused imports

### LINKED ISSUE #247: Implement autofix for `F401` (`UnusedImport`)

### PR DESCRIPTION
**Pull request by charliermarsh**
Resolves #247.

### CODE REVIEW

File: src/ast/types.rs [hunk 1]
```diff
@@ -61,6 +61,7 @@ pub enum BindingKind {
 pub struct Binding {
     pub kind: BindingKind,
     pub location: Location,
+    pub end_location: Location,
```
Comment (Seamooo) [inline src/ast/types.rs:64]: very exciting!

Comment (charliermarsh) [reply src/ast/types.rs:64]: I don't know if the spans will always be "correct" (that is: I can't guarantee they'll match the CPython semantics), so we may need to iterate on them a bit.

Comment (charliermarsh) [reply src/ast/types.rs]: (These fields are now available on `Message` and `Check`.)

**Comment by charliermarsh** [comment]
This works well but still requires a lot of bookkeeping around the AST manipulation, even with the help of LibCST.

**Pull request merged**